docs/source/create/avatars/avatar-standards.md

```diff
@@ -170,77 +170,79 @@ ReadyPlayerMe blendshapes are converted into Vircadia blendshapes automatically
 
 **Eye Blendshapes**
```

Comment:
I would suggest a starting paragraph explaining, something along the lines:
### Supported Blendshape names
I order to drive a Blendshape (on avatars or models) it has to either match one of the names in the list below, or use an *.fst file to remap it to one of the names in the list as described above.

Reply:
I've included an additional section titled Unsupported Blendshapes containing the information you suggested to include in the documentation.